cmd/contractcmd/init_validator_manager.go

```diff
@@ -154,6 +158,23 @@ func initValidatorManager(_ *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	logLvl, err := logging.ToLevel(validatorManagerFlags.aggregatorLogLevel)
+	if err != nil {
+		logLvl = logging.Debug
+	}
+	aggregatorLogDir := app.GetLogDir()
```

Comment:
nit: we use the same code in multiple places. maybe it makes sense to move it to something like `func (app *Avalanche) GetAggregatorLogger` which accepts clusterName for example and uses `aggregatorLogDir = app.GetLocalDir(clusterName)` for non-empty clusterName and `app.GetLogDir()` elsewhere